src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingWorkCoordinator.UnitTestingHighPriorityProcessor.cs

```diff
@@ -95,11 +96,13 @@ public void Enqueue(UnitTestingWorkItem item)
                             return;
                         }
 
+#if false // Not used in unit testing crawling
```

Comment:
Looks like this is nested inside another #if false and can potentially be omitted

Reply:
will do.